tools/api-markdown-documenter/src/RenderMarkdown.ts

```diff
@@ -68,14 +52,11 @@ export interface RenderDocumentsAsMarkdownOptions
  *
  * @param documents - The documents to render. Each will be rendered to its own file on disk per
  * {@link DocumentNode.documentPath} (relative to the provided output directory).
- * @param renderConfig - Configuration for rendering {@link DocumentNode}s as Markdown.
```

Comment:
All of this context is covered by property docs in `RenderDocumentsAsMarkdownOptions`.